ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -132,6 +130,13 @@ private RequestMapper createRequestMapper(
     final PassThroughHandler defaultHandler =
         new PassThroughHandler(downStreamConnection, transmitterFactory);
 
+    final VertxNonceRequestTransmitterFactory nonceRequestTransmitterFactory =
+        new VertxNonceRequestTransmitterFactory(
+            vertx.createHttpClient(clientOptions), jsonDecoder, httpRequestTimeout);
```

Comment:
better to reuse the same http client rather than create a new one since it has the same client options

Reply:
oh yeah - cut and paste ... missed that - nice.